src/MemoryTracing/MemoryTracingUtilsTest.cpp

```diff
@@ -161,17 +132,16 @@ SwapCached:      0 kB)",
 
   const std::string kEmptyMeminfo = "";
 
-  SystemMemoryUsage system_memory_usage;
-  ResetSystemMemoryUsage(&system_memory_usage);
+  SystemMemoryUsage system_memory_usage = CreateAndInitializeSystemMemoryUsage();
```

Comment:
as I said before, I'd rather prefer to have these sub-tests divided by "{", "}"

```
{
  SystemMemoryUsage system_memory_usage = CreateAndInitializeSystemMemoryUsage();
  GetValuesFromMemInfo(kValidMeminfo, &amp;system_memory_usage);  
 ExpectSystemMemoryUsageEq(system_memory_usage, kMemTotal, kMemFree, kMemAvailable, kBuffers,                            kCached);
}


{
  SystemMemoryUsage system_memory_usage = CreateAndInitializeSystemMemoryUsage();  
  GetValuesFromMemInfo(kPartialMeminfo, &amp;system_memory_usage);  
  ExpectSystemMemoryUsageEq(system_memory_usage, kMemTotal, kMemFree);
} 

```
.. etc

Reply:
Thanks for the suggestion! Changed accordingly.